actor/cfnetworkingaction/policy.go

```diff
@@ -0,0 +1,184 @@
+	if err != nil {
+		return []Policy{}, allWarnings, err
+	}
+
+	var v1Policies []cfnetv1.Policy
+	v1Policies, err = actor.NetworkingClient.ListPolicies()
+	if err != nil {
+		return []Policy{}, allWarnings, err
+	}
+
+	var policies []Policy
+	emptyPolicy := Policy{}
+	for _, v1Policy := range v1Policies {
+		policy := transformPolicy(applications, v1Policy)
+		if policy != emptyPolicy {
+			policies = append(policies, transformPolicy(applications, v1Policy))
+		}
+	}
+
+	return policies, allWarnings, nil
+}
+
+func (actor Actor) NetworkPoliciesBySpaceAndAppName(spaceGUID string, srcAppName string) ([]Policy, Warnings, error) {
+	var allWarnings Warnings
+	var appGUID string
+
+	applications, warnings, err := actor.V3Actor.GetApplicationsBySpace(spaceGUID)
+	allWarnings = append(allWarnings, Warnings(warnings)...)
+	if err != nil {
+		return []Policy{}, allWarnings, err
+	}
+
+	var v1Policies []cfnetv1.Policy
+
+	srcApp, warnings, err := actor.V3Actor.GetApplicationByNameAndSpace(srcAppName, spaceGUID)
+	allWarnings = append(allWarnings, Warnings(warnings)...)
+	if err != nil {
+		return []Policy{}, allWarnings, err
+	}
+
+	appGUID = srcApp.GUID
+	v1Policies, err = actor.NetworkingClient.ListPolicies(appGUID)
+	if err != nil {
+		return []Policy{}, allWarnings, err
+	}
+
+	var policies []Policy
+	emptyPolicy := Policy{}
+	for _, v1Policy := range v1Policies {
+		if appGUID == "" || v1Policy.Source.ID == appGUID {
+			policy := transformPolicy(applications, v1Policy)
+			if policy != emptyPolicy {
+				policies = append(policies, transformPolicy(applications, v1Policy))
+			}
+		}
+	}
+
+	return policies, allWarnings, nil
+}
+
+func (actor Actor) RemoveNetworkPolicy(spaceGUID, srcAppName, destAppName, protocol string, startPort, endPort int) (Warnings, error) {
+	var allWarnings Warnings
+
+	srcApp, warnings, err := actor.V3Actor.GetApplicationByNameAndSpace(srcAppName, spaceGUID)
+	allWarnings = append(allWarnings, Warnings(warnings)...)
+	if err != nil {
+		return allWarnings, err
+	}
+
+	destApp, warnings, err := actor.V3Actor.GetApplicationByNameAndSpace(destAppName, spaceGUID)
+	allWarnings = append(allWarnings, Warnings(warnings)...)
+	if err != nil {
+		return allWarnings, err
+	}
+
+	policyToRemove := cfnetv1.Policy{
+		Source: cfnetv1.PolicySource{
+			ID: srcApp.GUID,
+		},
+		Destination: cfnetv1.PolicyDestination{
+			ID:       destApp.GUID,
+			Protocol: cfnetv1.PolicyProtocol(protocol),
+			Ports: cfnetv1.Ports{
+				Start: startPort,
+				End:   endPort,
+			},
+		},
+	}
+
+	v1Policies, err := actor.NetworkingClient.ListPolicies(srcApp.GUID)
+	if err != nil {
+		return allWarnings, err
+	}
+
+	for _, v1Policy := range v1Policies {
+		if v1Policy == policyToRemove {
+			return allWarnings, actor.NetworkingClient.RemovePolicies([]cfnetv1.Policy{policyToRemove})
+		}
+	}
+
+	return allWarnings, PolicyDoesNotExistError{}
+}
+
+func transformPolicy(applications []v3action.Application, v1Policy cfnetv1.Policy) Policy {
```

Comment:
the `transformPolicy` method should be defined on the actor so that it is not floating in the package.